package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "crisper",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Since this is a new feature, this should be 2.1.0, but @usergenic can handle that.